pyracf/common/security_admin.py

```diff
@@ -20,6 +22,7 @@ class SecurityAdmin:
     _valid_segment_traits = {}
     _extracted_key_value_pair_segment_traits_map = {}
     _case_sensitive_extracted_values = []
+    __running_userid = False
```

Comment:
This should default to `None`, no?

Reply:
Yes, good catch. I had originally established this with `False` having a role in this process and this is a relic that was missed in the transition to `None`. Changed.